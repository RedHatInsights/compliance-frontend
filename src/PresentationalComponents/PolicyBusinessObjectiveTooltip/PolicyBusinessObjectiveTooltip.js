import React, { Fragment } from 'react';
import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
import { Tooltip } from '@patternfly/react-core';

const PolicyBusinessObjectiveTooltip = () => (
  <Tooltip
    position="right"
    content={
      <Fragment>
        This is an optional field that can be used to label policies that are
        related to specific business objectives.
      </Fragment>
    }
  >
    <span>
      &nbsp;
      <OutlinedQuestionCircleIcon className="grey-icon" />
    </span>
  </Tooltip>
);

export default PolicyBusinessObjectiveTooltip;
