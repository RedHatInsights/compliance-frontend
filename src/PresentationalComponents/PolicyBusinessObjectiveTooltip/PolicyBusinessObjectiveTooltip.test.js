import PolicyBusinessObjectiveTooltip from './PolicyBusinessObjectiveTooltip';

describe('PolicyBusinessObjectiveTooltip', () => {
    it('expect to render without error', () => {
        const wrapper = shallow(
            <PolicyBusinessObjectiveTooltip />
        );

        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
