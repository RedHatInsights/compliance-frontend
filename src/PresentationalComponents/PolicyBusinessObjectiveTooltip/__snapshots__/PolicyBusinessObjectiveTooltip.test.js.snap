// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PolicyBusinessObjectiveTooltip expect to render without error 1`] = `
<Tooltip
  content={
    <React.Fragment>
      This is an optional field that can be used to label policies that are related to specific business objectives.
    </React.Fragment>
  }
  position="right"
>
  <span>
    Â 
    <OutlinedQuestionCircleIcon
      className="grey-icon"
      color="currentColor"
      noVerticalAlign={false}
      size="sm"
    />
  </span>
</Tooltip>
`;
